### PR TITLE
modify views because of test warnings, refactor cart model test

### PR DESCRIPTION
**Pull request by gmichokostas**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -216,7 +216,7 @@ GEM
     erubis (2.7.0)
     eventmachine (1.0.4)
     execjs (2.2.2)
-    eye (0.6.3)
+    eye (0.6.4)
```
Comment (gmichokostas) [inline Gemfile.lock]: Hi @marcaltmann, I update eye gem as you said

File: README.rdoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ Please help us out by contributing code.
 
 === Installing Ruby and Ruby on Rails
 
-If you are not sure how to <em>install Ruby and the Ruby on Rails framework</em> on your computer, have a look at Michael Hartl's excellent {tutorial}[http://ruby.railstutorial.org/ruby-on-rails-tutorial-book#sec-rubygems] on the subject.
+If you are not sure how to <em>install Ruby and the Ruby on Rails framework</em> on your computer, have a look at Michael Hartl's excellent {tutorial}[http://rails-4-0.railstutorial.org/book/beginning#sec-rubygems] on the subject.
```
Comment (gmichokostas) [inline README.rdoc]: fix broken link

File: app/controllers/welcome_controller.rb [hunk 3]
```diff
@@ -37,7 +37,7 @@ def feed
     end
   end
 
-private
+  private
```
Comment (gmichokostas) [inline app/controllers/welcome_controller.rb]: rubocop was complaining

**Comment by gmichokostas** [comment]
sorry to be late, I was very busy all day long, thanks.

**Comment by marcaltmann** [comment]
No problem, thanks for your contribution.

**Pull request merged**